### PR TITLE
Get issue.json using axios

### PR DESCRIPTION
**Pull request by geckotang**
I often use [axios](https://github.com/mzabriskie/axios) for ajax.
Please kindly review this PRs.

### CODE REVIEW

File: lib/components/form.js [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
+    }
+  },
+  componentDidMount() {
+    let { account, repository } = this.props.params
+    let json = `${endpoints.rawgit}${account}/${repository}/master/issue.json`
+    this.setState({
+      account: account,
+      repository: repository
+    })
+    this.serverRequest = request.get(json).then( (res) => {
```
Comment (watilde) [inline lib/components/form.js:27]: How about to use `$.get` instead of axios?
https://facebook.github.io/react/tips/initial-ajax.html

Comment (geckotang) [inline lib/components/form.js:27]: @watilde what is `$` on the react tips? I guess the tips said you could use `$` when you already use jQuery in your project. Thus we can use any library for Ajax and then I use axios.

Comment (watilde) [inline lib/components/form.js:27]: 100% correct! nvm please.

**Comment by watilde** [comment]
LGTM! You can merge this :)

**Comment by watilde** [comment]
Thanks arigato!

**Pull request merged**